src/teaching-element/Assessment/SingleChoice.vue

```diff
@@ -36,20 +38,30 @@ export default {
     return { userAnswer: this.submission };
   },
   computed: {
+    mappedAnswers() {
+      return this.answers.map((answer, index) => ({
```

Comment:
The index is not used so it can be removed.

Reply:
👍